test-end-to-end/android-maestro/common/add_6_students_by_bulk_import.yaml

```diff
@@ -0,0 +1,30 @@
+appId: com.toughra.ustadmobile
+
+---
+# This test tests the Library page of app adding content
+# Test parameters needs to provide for this test :
+ # 1) TESTFILENAME : provide the content name inside the 'test-files/content' folder
```

Comment:
Why does this have to be add 6 students? Can't this just be any bulk upload CSV? The parameter names need to be updated. TESTFILENAME should be provide the name of the "bulk user CSV file" not "content name".

Reply:
student_bulk_upload_csv.yaml renamed.